### PR TITLE
fix(DB/CreaturesSAI): Mobs Casting Daze

### PR DESCRIPTION
**Pull request by Stifler82**
## Changes Proposed:
-  I did a "Select" of all NPCs who had SAI with "Cast" and Spell "Daze", spell Daze **(in any case) MUST be cast from behind** and **NEVER in front**, so with this Fix, **I fixed ALL the NPCs of the World who gave this problem**..

## Issues Addressed:
- Closes https://github.com/chromiecraft/chromiecraft/issues/61
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4571

## Tests Performed:
- Tested in game and working
- Tested on Windows 10 (64)
- **Video Test:** [Mobs casting daze](https://youtu.be/feU1B9hq_-o)

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Stifler82** [comment]
@FrancescoBorzi
@Helias

**Comment by FALL1N1** [comment]
Isn't Dazed supposed to be handled within the core in the first place?

**Comment by Stifler82** [comment]
> Isn't Dazed supposed to be handled within the core in the first place?

@FALL1N1 
I have already answered your question here: https://github.com/chromiecraft/chromiecraft/issues/61#issuecomment-785151010

**I wrote this:**
```
however I want to clarify:
That **this "Daze"** isn't the **"Daze - Global - Game Mechanics"**,
this Daze that some Mobs had and used it in front of the Player, **was a "Spell"**,
therefore a **different "Daze"**, in fact the **Daze (Of global mechanics )** 
has a **duration of 5sec**, while **the Daze of these mobs has a duration of 9sec**.
```

**Review by Helias — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
no need to test again as the author provided evidence of testing (video)

File: data/sql/updates/pending_db_world/rev_1614138891078013900.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1614138891078013900');
+
+UPDATE `smart_scripts` SET `event_type` =67, `event_param1` =15000, `event_param2` =15000, `event_flags` =0 WHERE `entryorguid` IN (3,36,92,114,822,1128,1199,1201,1537,1543,1547,1548,1549,1770,1869,1934,1935,1972,2248,2347,2473,2474,2723,2735,2736,2780,2781,2782,4130,4872,5246,5253,5292,5293,5300,5461,5850,5853,7032,8447,9218,9219,11698,15655) AND (`source_type` =0);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1614138891078013900.sql:3]: @Stifler82 shouldn't we also include the following to the `WHERE` ?

```sql
AND `action` = 11 -- action Cast
AND `action_param1` = 13496 -- spell 'Dazed`
```

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by pklloveyou** [comment]
Tested

**Pull request merged**